src/features/content-explorer/content-explorer/__tests__/ContentExplorerEmptyState.test.js

```diff
@@ -19,5 +19,17 @@ describe('features/content-explorer/content-explorer/ContentExplorerEmptyState',
 
             expect(wrapper.find('SearchEmptyState').length).toBe(1);
         });
+
+        test('should not render any text when isSearch is false, isIncludeSubfolders is true, and isOnInitialModalPage is true', () => {
```

Comment:
We might want to test all possible combinations with `withData` to make sure conditional statements in `ContentExplorerEmptyState` are protected from mistakes.